src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
+
+export const routes: Routes = [];
```

Comment:
document

          export const routes: Routes = [];

          Explantion: The routes constant initializes an empty array. Since no routes are defined, consider adding documentation to clarify that this is intentional.